lib/admin.tsx

```diff
@@ -1,3 +1,39 @@
+export async function getProfiles(query) {
```

Comment:
@jzeiders worked on some new client fetcher api, to be honest i'm ok w/ this for now, tho we might want to port this over later

Reply:
I'll tag it as a todo